src/commcare_cloud/ansible/roles/bootstrap-users/tasks/main.yml

```diff
@@ -26,9 +36,24 @@
 
 - name: Add dev group
   become: yes
-  group: name="{{ dev_group }}" state=present
+  group:
+    name: "{{ dev_group }}"
+    state: present
```

Comment:
This change is reformatting only, correct? It would be nice if changes like this were in a different commit from the uid/gid additions.

Reply:
Correct, it was just to uniform the format and make it more readable. I can split the commits.

Reply:
Your choice at this point. It would have made it easier to review.

Reply:
I've split the commits and changed the default uid and gid values.